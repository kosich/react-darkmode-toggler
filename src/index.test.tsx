import React from 'react'
import { shallow } from 'enzyme'
// Import Components
import { DarkToggler } from './index'
const clickFn: any = jest.fn()

describe('first', () => {
  test('Should render two ', () => {
    const component = shallow(<DarkToggler isDark={false} onClick={clickFn} />)
    component.find('input#checkbox').simulate('click')
    expect(component).toMatchSnapshot()
  })
})
