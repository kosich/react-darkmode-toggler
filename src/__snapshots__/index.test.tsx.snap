// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`first Should render two  1`] = `
<div
  className="container small"
>
  <label
    className="switch"
    htmlFor="checkbox"
    title="Change color scheme to dark mode"
  >
    <input
      checked={false}
      id="checkbox"
      onChange={[MockFunction]}
      type="checkbox"
    />
    <div
      className="slider round"
    >
       
    </div>
    <div
      className="hide"
    >
      🌙
    </div>
    <div
      className="togglesun"
    >
      ☀️
    </div>
  </label>
</div>
`;
